__tests__/app.test.js

```diff
@@ -45,25 +37,102 @@ describe('/api', () => {
     });
 });
 
-describe('/api/articles/:article_id', () => {
+describe('GET /api/articles/:article_id', () => {
     test('GET: 200 an article object with all associated properties', () => {
         return request(app)
         .get('/api/articles/1')
         .expect(200)
         .then((response) => {
-            const {rows} = response.body
-            rows.forEach(row => {
-                expect(row).toMatchObject({
+            const {article} = response.body
+            expect(article.article_id).toBe(1)
+            expect(article).toMatchObject({
+                created_at: expect.any(String)
+            })
+        })
+    });    
+});
+
+describe('GET /api/articles', () => {
+    test('GET: 200 returns an array of all objects within articles without body property, sorted in descending order by creation date', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const {articles} = response.body
+            expect(articles.length).toBe(13)
+            articles.forEach(article => {
+                expect(article).toMatchObject({
                     title: expect.any(String),
                     topic: expect.any(String),
                     author: expect.any(String),
-                    body: expect.any(String),
                     created_at: expect.any(String),
                     votes: expect.any(Number),
-                    article_img_url:
-                    expect.any(String),
+                    article_img_url: expect.any(String),
+                    comments_count: expect.any(String)
```

Comment:
Nice, you may wish to check the comment count for one article, just to check the comment count is correct :)

Reply:
We should also be testing that it doesn't include the body property anymore.
https://jestjs.io/docs/expect#expectnotobjectcontainingobject